sentry/worker.py

```diff
@@ -5,37 +5,13 @@
 from datetime import timedelta
 
 import sentry_sdk
-from temporalio import activity, workflow
+from sentry.workflows import compose_greeting, GreetingWorkflow
```

Comment:
```suggestion
from sentry.activities import compose_greeting
from sentry.workflows import GreetingWorkflow
```

Should import from the canonical place the thing is defined IMO